test/functional/interface_bitcoin_cli.py

```diff
@@ -66,34 +105,30 @@ def run_test(self):
+        cli_get_info = cli_get_info_string_to_dict(cli_get_info_string)
+
         network_info = self.nodes[0].getnetworkinfo()
         blockchain_info = self.nodes[0].getblockchaininfo()
-        assert_equal(cli_get_info['version'], network_info['version'])
-        assert_equal(cli_get_info['blocks'], blockchain_info['blocks'])
-        assert_equal(cli_get_info['headers'], blockchain_info['headers'])
-        assert_equal(cli_get_info['timeoffset'], network_info['timeoffset'])
-        assert_equal(
-            cli_get_info['connections'],
-            {
-                'in': network_info['connections_in'],
-                'out': network_info['connections_out'],
-                'total': network_info['connections']
-            }
-        )
-        assert_equal(cli_get_info['proxy'], network_info['networks'][0]['proxy'])
-        assert_equal(cli_get_info['difficulty'], blockchain_info['difficulty'])
-        assert_equal(cli_get_info['chain'], blockchain_info['chain'])
+        assert_equal(int(cli_get_info['Version']), network_info['version'])
+        assert_equal(int(cli_get_info['Blocks']), blockchain_info['blocks'])
+        assert_equal(int(cli_get_info['Headers']), blockchain_info['headers'])
+        assert_equal(int(cli_get_info['Time offset (s)']), network_info['timeoffset'])
+        expected_network_info = f"in {network_info['connections_in']}, out {network_info['connections_out']}, total {network_info['connections']}"
+        assert_equal(cli_get_info["Network"], expected_network_info)
+        assert_equal(cli_get_info['Proxy'], network_info['networks'][0]['proxy'])
+        assert_equal(Decimal(cli_get_info['Difficulty']), blockchain_info['difficulty'])
+        assert_equal(cli_get_info['Chain'], blockchain_info['chain'])
 
         if self.is_wallet_compiled():
             self.log.info("Test -getinfo and bitcoin-cli getwalletinfo return expected wallet info")
-            assert_equal(cli_get_info['balance'], BALANCE)
-            assert 'balances' not in cli_get_info.keys()
+            assert_equal(float(cli_get_info['Balance']), BALANCE)
+            assert 'Balances' not in cli_get_info_string
             wallet_info = self.nodes[0].getwalletinfo()
-            assert_equal(cli_get_info['keypoolsize'], wallet_info['keypoolsize'])
-            assert_equal(cli_get_info['unlocked_until'], wallet_info['unlocked_until'])
-            assert_equal(cli_get_info['paytxfee'], wallet_info['paytxfee'])
-            assert_equal(cli_get_info['relayfee'], network_info['relayfee'])
+            assert_equal(int(cli_get_info['Keypool size']), wallet_info['keypoolsize'])
+            assert_equal(int(cli_get_info['Unlocked until']), wallet_info['unlocked_until'])
+            assert_equal(Decimal(cli_get_info['Pay transaction fee (BTC/kvB)']), wallet_info['paytxfee'])
+            assert_equal(Decimal(cli_get_info['Relay fee (BTC/kvB)']), network_info['relayfee'])
```

Comment:
```suggestion
            assert_equal(Decimal(cli_get_info['Min tx relay fee (BTC/kvB)']), network_info['relayfee'])
```
(see the getnetworkinfo help)

Reply:
Agreed. Updated to `Relay fee` to `Min tx relay fee rate`